sampleconfig/core.yaml

```diff
@@ -53,6 +53,9 @@ peer:
         # endorsementTimeout is the duration the gateway waits for a response
         # from other endorsing peers before returning a timeout error to the client.
         endorsementTimeout: 30s
+        # broadcastTimeout is the duration the gateway waits for a response
+        # from ordering nodes before returning a timeout error to the client.
+        broadcastTimeout: 30s
```

Comment:
This is a really long default timeout before failing over to another node. The legacy SDKs used 3 seconds. For a node deployed fairly locally on the network, even that is pretty long to wait for a remote call